cmd/ddev/cmd/service-enable.go

```diff
@@ -13,7 +14,7 @@ const disabledServicesDir = ".disabled-services"
 var ServiceEnable = &cobra.Command{
 	Use:     "enable service [project]",
 	Short:   "Enable a 3rd party service",
-	Long:    fmt.Sprintf(`Enable a 3rd party service. The service must exist as .ddev/%s/docker-compose.<service>.yaml. Note that you can use "ddev get" to obtain a service not already on your system.`, disabledServicesDir),
+	Long:    fmt.Sprintf(`Enable a 3rd party service. The service must exist as .ddev/%s/docker-compose.<service>.yaml. Note that you can use "ddev add-on get" to obtain a service not already on your system.`, disabledServicesDir),
```

Comment:
I think we should probably go ahead and deprecate `ddev service` at this point. 
* https://github.com/ddev/ddev/issues/6298
Would it be reasonable to do that in this PR or not?

Reply:
I'd rather keep this PR down to just what it needs to touch (especially since there's a separate issue for that work already). There's a lot of room for scope creep here but it'll be easiest to work on and to review if we just keep it narrow, and do separate work separately.